src/frontend/shared-components/src/__snapshots__/storyshots.test.js.snap

```diff
@@ -1866,6 +1866,463 @@ exports[`Storyshots Loader Page 1`] = `
 </Loader>
 `;
 
+exports[`Storyshots Table With Blue Stripe Style 1`] = `
+<Table
+  table={
+    Object {
+      "header": "BCeID Info",
+      "tableElements": Array [
+        Object {
+          "name": "BCeID",
+          "value": "bobross42",
+        },
+        Object {
+          "name": "Last Name",
+          "value": "Ross",
+        },
+        Object {
+          "name": "First Name",
+          "value": "Bob",
+        },
+        Object {
+          "name": "Email Address",
+          "value": "bob.ross@pbs.com",
+        },
+      ],
+      "tableStyle": "blueStripe",
+    }
+  }
+>
+  <table
+    className="blueStripe"
```

Comment:
nit: ive been using this type of syntax for classnames: `blue-stripe` if we care about making it similar for all components that might be nice to have the consistency for someone else consuming it